docs/es/docs/tutorial/response-status-code.md

```diff
@@ -0,0 +1,105 @@
+
+En HTTP, tu envías un código de estado de tres dígitos como parte de la respuesta.
+
+Estos códigos de estado tienen un nombre asociado para reconocerlos, pero lo más importante es el número.
+
+En resumen:
+
+ - `100` en adelante son para "Información". Rara vez los usas directamente. Las respuestas con estos códigos de estado no tienen body.
+ - `200` en adelante son para respuestas "Exitosas". Esta son las que se usan con mayor frecuencia.
+      - `200` código  de estado por defecto, el cual significa que todo estuvo "OK".
+      - Otro ejemplo sería `201`, "Creado". Es comúnmente usado después de crear una observación en la base de datos.
+      - `204` es un caso especial, "Sin contenido". Esta respuesta es usada cuando no hay contenido de retorno por parte del cliente,
+      y la respuesta no debe tener cuerpo.
+
+- `300` en adelante son para "Redirección". Respuestas con estos códigos de estado pueden o no tener body, excepto para `304`, "No Modificado",
+  el cual no debe tener body.
+
+- `400` en adelante son para respuestas para `Error del Cliente`. Estos son el segundo tipo de codigós que más usarás.
+     - Un ejemplo es `404`, para respuesta de "No encontrado".
+     - Para errores genéricos desde el cliente, puedes usar `400`.
+
+- `500` en adelante son para errores del servidor. Tú nunca los usas directamente. Cuando algo sale mal en alguna parte del código de tu aplicación,
+  o en el servidor, automáticamente retornará uno de estos códigos.
+
+!!! tip
+    Para saber más acerca de estos códigos de estado y cuál código se usa para cada caso, revisa la
+<a href="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status" class="external-link" target="_blank"><abbr title="Mozilla Developer Network"> documentación acerca de los códigos de estados MDN</abbr> .
+
+# Atajos para recordar los nombres
+
+Vamos a ver el ejemplo previo otra vez:
+
+```Python hl_lines="6"
+{!../../../docs_src/response_status_code/tutorial001.py!}
+```
+
+`201` es el código de estado para "Creado".
+
+Pero tú no tienes que memorizar lo que significa cada uno de estos códigos.
+
+Puedes usar las convenientes variables desde `fastapi.status`.
+
+```Python hl_lines="6"
+{!../../../docs_src/response_status_code/tutorial002.py!}
+```
+
+Son tan convenientes, mantienen el mismo número, pero puedes usar el autocompletado del editor para encontrarlas:
+
+<img src="../../../en/docs/img/tutorial/response-status-code/image02.png">
+
+!!! note "Detalles técnicos"
+    Podrías usar también `from starlette import status`.
+
+
+```
+**FastAPI** provee el mismo `starlette.status` como  `fastapi.status`  conveniente para tí, el desarrollador. Pero viene directamente de Starlette.
+```
+
+# Cambiando el por defecto
+
+Después, en la [Guía Avanzada de Usuario](https://github.com/carlosm27/fastapi/blob/master/docs/en/docs/advanced/response-change-status-code.md), verás cómo retornar diferentes códigos de estado que los de por defecto declaras aquí.
```

Comment:
```suggestion
Después, en la [Guía Avanzada de Usuario](https://github.com/carlosm27/fastapi/blob/master/docs/en/docs/advanced/response-change-status-code.md), verás cómo retornar un código de estado diferente al predeterminado que declaras aquí.
```

Reply:
Isn't there any problem with the link to the forked repo?